src/main/java/mate/academy/bookstore/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.bookstore.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.BookQuantityDto;
+import mate.academy.bookstore.dto.CartItemRequestDto;
+import mate.academy.bookstore.dto.ShoppingCartResponseDto;
+import mate.academy.bookstore.exception.EntityNotFoundException;
+import mate.academy.bookstore.mapper.CartItemMapper;
+import mate.academy.bookstore.mapper.ShoppingCartMapper;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.model.CartItem;
+import mate.academy.bookstore.model.ShoppingCart;
+import mate.academy.bookstore.model.User;
+import mate.academy.bookstore.repository.CartItemRepository;
+import mate.academy.bookstore.repository.ShoppingCartRepository;
+import mate.academy.bookstore.repository.book.BookRepository;
+import mate.academy.bookstore.service.ShoppingCartService;
+import mate.academy.bookstore.service.UserService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    private final CartItemRepository cartItemRepository;
+    private final ShoppingCartRepository shoppingCartRepository;
+    private final ShoppingCartMapper shoppingCartMapper;
+    private final CartItemMapper cartItemMapper;
+    private final BookRepository bookRepository;
+    private final UserService userService;
+
+    @Override
+    public ShoppingCart getShoppingCartModel() {
+        User user = userService.getUser();
+        return shoppingCartRepository.findById(user.getId()).get();
+    }
```

Comment:
This method should be private method and excluded from interface. It's inconsistent with others, the only one that returns model instead of DTO and as a result unused from outside